config/master/aws-k8s-cni-cn.yaml

```diff
@@ -112,6 +112,7 @@
             - "/app/grpc-health-probe"
             - "-addr=:50051"
           "initialDelaySeconds": 35
+          "periodSeconds": 60
```

Comment:
If the CNI pod starts up, we will always return `HealthCheckResponse_SERVING`. If the calls to EC2 API gets throttled, it can take a few minutes for the pod to come up. Increasing the period from the default of 10 seconds will give it some more time to get out of throttling and start up before it gets killed and restarted. 

Question: Is it better to just bump the initial delay? What is a good time between checks? By default, 3 failed checks are required before the pod gets killed and restarted.

Reply:
Right, it sounds like this disagrees with the [above](https://github.com/aws/amazon-vpc-cni-k8s/pull/1028#discussion_r441826154) `initialDelaySeconds` "the pod will get ready in just a few seconds" statement.

If it's only start time, then yes, I would increase `initialDelaySeconds` instead.  Increasing `periodSeconds` means that we're also going to take longer to respond to an issue _after_ startup, which is not great.

Note k8s 1.16 also has a new `startupProbe` which is similar to just increasing initialDelaySeconds, but overkill in this case imo.

Reply:
Yes, better to increase the initial delay. I agree that it sounds like two opposing statements, but I just think it's two different cases. It is true that ipamd usually starts fast, but not if it gets throttled by EC2. In that case we want to keep retrying without the pod getting killed before it is even ready.